app/services/api/jsonrpc/jsonrpc.ts

```diff
@@ -107,6 +116,7 @@ export class JsonrpcService extends Service implements IJsonrpcServiceApi {
     data: any;
     isRejected?: boolean;
   }): IJsonRpcResponse<IJsonRpcEvent> {
+    // eslint-disable-next-line
     return JsonrpcService.createResponse.apply(this, arguments);
```

Comment:
Curious why we need to disable so much in this file.

Reply:
Using of `arguments` is not allowed. Each disabled line contains the `arguments` keyword